.github/template_gitref

```diff
@@ -1 +1 @@
-2021.08.26-106-g556224f
+2021.08.26-159-ga96338a
```

Comment:
I would not expect this to change if the plugin_template was checked out to `556224f` when the script was run

Reply:
doh, yeah, that pulled in master plugin_template.  Fixing.